apps/system_message/resource.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.resource import Resource
+
+
+class SystemMessagesResource(Resource):
+    schema = {
+        "is_active": {"type": "boolean", "default": False},
+        "type": {
+            "type": "string",
+            "allowed": ["warning", "alert", "primary/info", "success"],
+            "required": True,
+        },
+        "message_title": {"type": "string", "required": True},
+        "message": {"type": "string", "required": True},
+        "user": {"type": "string", "required": True},
```

Comment:
what is this used for?

Reply:
Just in case, if we need it in the future to know who created the message.

Reply:
and is that a name or user id?

Reply:
it's the `user_id`. I think I got your point here, I'll change it to `user_id`